packages/map-template/src/components/Search/Categories/Categories.jsx

```diff
@@ -124,6 +126,25 @@ function Categories({ onSetSize, getFilteredLocations, searchFieldRef }) {
         });
     }
 
+    /**
+     * Get the active category element.
+     */
+    useEffect(() => {
+        if (selectedCategory) {
+            getActiveCategory().then(activeCategory => setActiveCategory(activeCategory));
```

Comment:
Can we listen for the categories atom instead of using a mutationobserver?

Reply:
I'm not sure, because we need to determine which one is the active category, and that's why we're using the Mutation Observer here.

Reply:
Try playing around with making a conditional ref (or whatever one could call it) for the active chip.

In the JSX where we loop through the categories and present `<mi-chip>` elements, add something like

`const refProp = selectedCategory === category ? { ref: activeChipRef } : {};`

And add `{...refProp}` as an attribute to the `<mi-chip>` element.

Then it may (this is not tested!! 👈 ) be possible to listen for the appearance of the `activeChipRef` in a `useEffect` hook and do the scrolling there. Without the use of a MutationObserver.

Reply:
I tried your suggestion, and it doesn't seem to work because the selected category is undefined at the time when we render the categories. 

I tried to fiddle around with it by creating another `useState` and assigning that at the moment when we have the `selectedCategory` present, but didn't get it to work. I might be doing something wrong.

I stashed my changes, we can take a look at it together whenever you have some time, and see if we can get it to work.

Reply:
I just tried an actual implementation of my idea, and can't get it to work without introducing a `setTimeout` 😂 I think it may be solvable though (I am also a little confused about the difference between activeCategory and selectedCategory).

Reply:
Good old `setTimeout` 😂

Reply:
Should we see if we can get it to work though?

Reply:
After spending some time on fixing this, we decided to postpone a fix, and use what we have now.